dora/integration/fuse/src/main/java/alluxio/fuse/AlluxioFuse.java

```diff
@@ -210,34 +226,70 @@ public static FuseUmountable launchFuse(FileSystemContext fsContext, FileSystem
       }
     }
 
-    final boolean debugEnabled = conf.getBoolean(PropertyKey.FUSE_DEBUG_ENABLED);
     if (conf.getBoolean(PropertyKey.FUSE_JNIFUSE_ENABLED)) {
       final AlluxioJniFuseFileSystem fuseFs
           = new AlluxioJniFuseFileSystem(fsContext, fs, fuseOptions);
+      return fuseFs;
+    } else {
+      AlluxioJnrFuseFileSystem fuseFs = new AlluxioJnrFuseFileSystem(fs, conf, fuseOptions);
+      return fuseFs;
+    }
+  }
 
+  /**
+   * Preprocess the instance of {@link AlluxioJniFuseFileSystem} before mounting.
+   *
+   * @param jniFuseFileSystem the instance of {@link AlluxioJniFuseFileSystem}
+   */
+  public void preprocess(AlluxioJniFuseFileSystem jniFuseFileSystem) {
```

Comment:
this naming is a bit unnatural to me. Can we rename this to something like setupFuseFileSystem?

Reply:
So, this function is empty and can be overrided to add special handlings?

Reply:
> So, this function is empty and can be overrided to add special handlings?
Actually, this method can be regarded as a hook method, and we can override it in the subclass of AlluxioFuse (e.g. the EE class).

Reply:
> this naming is a bit unnatural to me. Can we rename this to something like setupFuseFileSystem?

Sure, `setupFuseFileSystem` is better. The method is for the action before startup.